HW_04_Task1/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace HW_04_Task1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            for (int i = 0; i < 26; i++)

```

Comment:
26, 90 - these are magic numbers for me. I would suggest to define a local variables for these numbers